### PR TITLE
Add SHARK token for Shark Protocol

### PR DESCRIPTION
**Pull request by racoon-supply-rac**
## Description

Adding token: SHARK from chain Migaloo

### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] Ready -- A liquidity pool has been created. The pool ID is: ______
- [x] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [x] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Please do not edit the osmosis-1.assetlist directly. Only add to zone_assets.

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -2548,6 +2548,12 @@
       "base_denom": "factory/osmo1279xudevmf5cw83vkhglct7jededp86k90k2le/RAPTR",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+        {
+      "chain_name": "migaloo",
+      "base_denom": "factory/migaloo1eqntnl6tzcj9h86psg4y4h6hh05g2h9nj8e09l/shark",
+      "path": "transfer/channel-642/factory/migaloo1eqntnl6tzcj9h86psg4y4h6hh05g2h9nj8e09l/shark",
+      "osmosis_verified": false
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2556]: Please add "osmosis_unlisted": true

**Comment by JeremyParish69** [comment]
When is the Stream date?

**Comment by racoon-supply-rac** [comment]
> When is the Stream date?

Early March, would you prefer we open a pool before the StreamSwap? We wanted to use the StreamSwap funds to open a pool. Let me know

**Comment by JeremyParish69** [comment]
> > When is the Stream date?
> 
> Early March, would you prefer we open a pool before the StreamSwap? We wanted to use the StreamSwap funds to open a pool. Let me know

It's typical for teams launching via StreamSwap to use the funds to create their first pool, but in that case we won't enlist the asset until the Stream has begun.

**Comment by racoon-supply-rac** [comment]
> > > When is the Stream date?
> > 
> > 
> > Early March, would you prefer we open a pool before the StreamSwap? We wanted to use the StreamSwap funds to open a pool. Let me know
> 
> It's typical for teams launching via StreamSwap to use the bootstrapping funds to create their first pool, but in that case we won't enlist the asset until the Stream has begun.

Will it be visible on StreamSwap? I spoke with them and they said it should be visible on Osmosis to be seen on StreamSwap. I can re-confirm with them. Thanks

**Comment by JeremyParish69** [comment]
> Will it be visible on StreamSwap? I spoke with them and they said it should be visible on Osmosis to be seen on StreamSwap. I can re-confirm with them. Thanks

As far as I'm aware, their token listing process is separate and doesn't have any strict technical dependency on its visibility on Osmosis Zone. To this point, I think it's only been important (for better UX) that the token is visible on Osmosis before the Stream ends.

**Comment by JeremyParish69** [comment]
I see SHARK is now in a Bootstrapping phase. Are we planning on adding the token to Osmosis now? Perhaps in another PR?

**Pull request closed without merging**